src/js/view/configurations/configureStore.test.js

```diff
@@ -0,0 +1,23 @@
+import { configureStore } from './configureStore';
+
+it('should saveStateToDisk', () => {
+  const fakeStore = {
+    getState: () => {
+      console.log('getState invoked');
+      return {
+        logInfoReducer: 'hejhoj',
+        logViewerReducer: 'one',
+        testStuff: 'two'
+      };
+    }
+  };
+  const fakeRequester = params => {
+    expect(params).toEqual({
```

Comment:
If fakeRequester is never called by configureStore then this test passes. Shouldn't it fail?

reproduce: 
Comment out all the code in saveStateToDisk and run tests

Reply:
Korrekt, ett bättre sätt borde vara att ersätta med en https://jestjs.io/docs/en/mock-functions som failar testet om den inte blivit anropad med rätt payload.